src/main/java/com/grum/geocalc/BoundingArea.java

```diff
@@ -131,4 +139,72 @@ public int hashCode() {
         hash = 13 * hash + (this.southWest != null ? this.southWest.hashCode() : 0);
         return hash;
     }
+    
+    /**
+     * Get North Latitude
+     * @return double latitude
+     */
+    public double getNorthLatitude() {
+        return northEast.getLatitude();
+    }
```

Comment:
this and al. are only used in the same class (and associated tests) and do not bring much value. If anything, that clutters  the code quite a bit.